addons/config/templates/cpi.tanzu.vmware.com/v1alpha1/vspherecpiconfig.yaml

```diff
@@ -8,3 +8,16 @@ metadata:
 spec:
   vsphereCPI:
     mode: #@ "vsphereCPI" if data.values.IAAS != "tkgs" else "vsphereParavirtualCPI"
+---
```

Comment:
Question: Appreciate if there are some explanations in issue description or inline comments on why we need two separate `VSphereCPIConfig` CRs in the case of TKG on vSphere(TKGs) ?

Reply:
Hi Danniel! Fixed the issue description in PR. We want a routable specific VsphereCPIConfig for enabling routable pod feature. Since the routable pod feature is optional in tkgs, we want two separate VsphereCPIConfig for that.